jfrog_platform/README.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Overview
+The following describes how to configure Datadog to gather metrics from JFrog Artifactory and Xray.
+
+### What is JFrog Artifactory and Xray
+JFrog Enterprise with Xray features Artifactory Enterprise and Xray. Together, they empower DevOps teams to improve their productivity to increase velocity and deliver high quality software releases with confidence. Xray scans for known open source security risks and compliance using the industry's most comprehensive intelligence, VulnDB.
+Artifactory supports multiple build packages, artifacts, and metadata. It allows DevOps teams to have the freedom of choice of build packages (Bower, Chef, CocoaPods, Conan, Conda, CRAN, Debian, Docker, Golang, Gradle, Git LFS, Helm, Ivy, Maven, npm, NuGet, Opkg, P2, PHP Composer, Puppet, PyPI, RPM, RubyGems, SBT, Vagrant & VCS), CI/CD platforms, and DevOps tools.
+
+Enterprise supports multi-region, multi-cloud and hybrid replication for geographically distributed teams allowing you to replicate between a source repository and multiple targets simultaneously, and security features such as, IP filtering, CNAME, and data encryption at rest.
+
+Artifactory supports Kubernetes for microservices and containerized applications. Manage your deployments and gain insight into dependencies using Artifactory as your Kubernetes registry, while Xray provides container security scanning all layers recursively, ensuring that every artifact and dependency included in your Docker image has been scanned for known risks. Enterprise meets your business model needs supporting hybrid, cloud, and multi-cloud environments.
```

Comment:
```suggestion
Artifactory supports Kubernetes for microservices and containerized applications. Manage your deployments and gain insight into dependencies using Artifactory as your Kubernetes registry. AWS X-Ray provides container security scanning all layers recursively, ensuring that every artifact and dependency included in your Docker image has been scanned for known risks. Enterprise meets your business model needs supporting hybrid, cloud, and multi-cloud environments.
```

Reply:
Changed "Xray" to "JFrog Xray" in a separate commit.